frontend/src/components/crud/Create.tsx

```diff
@@ -0,0 +1,115 @@
+import React, { useState } from "react";
+import { JSONSchema7 } from "json-schema";
+import { Form } from "@rjsf/bootstrap-4";
+import CrudAPIClient from "../../APIClients/CrudAPIClient";
+
+const pythonSchema: JSONSchema7 = {
+  title: "Create Entity",
+  description: "A simple form to test creating an entity",
+  type: "object",
+  required: [
+    "string_field",
+    "int_field",
+    "string_array_field",
+    "enum_field",
+    "bool_field",
+  ],
+  properties: {
+    string_field: {
+      type: "string",
+      title: "String Field",
+      default: "UW Blueprint",
+    },
+    int_field: {
+      type: "integer",
+      title: "Integer Field",
+      default: 2017,
+    },
+    string_array_field: {
+      type: "array",
+      items: {
+        type: "string",
+      },
+      title: "String Array Field",
+      default: [],
+    },
+    enum_field: {
+      type: "string",
+      enum: ["A", "B", "C", "D"],
+      title: "Enum Field",
+      default: "B",
+    },
+    bool_field: {
+      type: "boolean",
+      title: "Boolean Field",
+      default: true,
+    },
+  },
+};
+
+// const typescriptSchema: JSONSchema7 = {
```

Comment:
Using this schema only should be fine (camelCase). The `BaseAPIClient` will automatically do a casing conversion when making the request if using the Python API 🙂